psi4/src/read_options.cc

```diff
@@ -489,7 +489,7 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     (the one-particle density matrices are written for all roots).
     Useful for a state-specific CASSCF or CI optimization on an
     excited state. -*/
-    options.add_int("FOLLOW_ROOT", 1);
+    options.add_int("FOLLOW_ROOT", 0);
```

Comment:
Also,
* https://github.com/psi4/psi4/blob/master/psi4/src/read_options.cc#L318
* https://github.com/psi4/psi4/blame/master/doc/sphinxman/source/glossary_psivariables.rst#L340


Can https://github.com/psi4/psi4/blob/master/psi4/src/read_options.cc#L2075 be changed cosmetically easily?

@fevangelista, https://github.com/psi4/psi4/blob/master/psi4/src/read_options.cc#L2283 is typically ground state, yes? Can it's external ordering be switched to 0?